platform-coop-toolkit/coop-library-framework
# Add date validation (resolve #71)

## Description

This PR ensures that date inputs are valid ISO 8601 strings. Do not merge until #84 has been merged.

## Steps to test

1. Input a date string that is not valid, e.g. `2019-20-20` or `abcdefg`.
2. Try to save your resource.

**Expected behavior:** Appropriate error messages are displayed.

## Additional information

- Uses the RegEx validation pattern from [jQuery Validator](https://github.com/jquery-validation/jquery-validation/).

## Related issues

- Resolves #71
